eq-author-api/utils/datastoreDynamo.js

```diff
@@ -21,6 +23,41 @@ const saveModel = (model, options = {}) =>
     });
   });
 
+const createUser = user => {
+  const { id, email, name, sub, picture } = user;
+  return saveModel(
+    new UserModel({
+      id: id ? id : uuid.v4(),
```

Comment:
Sometimes it is created with a known id?

```suggestion
      id: id || uuid.v4(),
```